src/pipeline_manager.py

```diff
@@ -410,6 +410,7 @@ def _read_data(dev_mode, read_train=True, read_test=False):
                                                                   on='SK_ID_BUREAU', how='right')
     if dev_mode:
```

Comment:
@kstrzala I don't understand what it does since we've loaded only nrows from bureau anyways via `.read_csv(...nrows)`